scripts/train/train.py

```diff
@@ -482,6 +474,15 @@ def main(cfg: DictConfig) -> Trainer:
         device_train_batch_size,
     )
 
+    # Callbacks
```

Comment:
I'm assuming you had to move this below the dataloader creation. Given that the ordering now matters, please leave behind a comment.

Reply:
resolved!